src/rules/at-rule-name-newline-after/README.md

```diff
@@ -0,0 +1,133 @@
+# at-rule-name-newline-after
+
+Require a newline after at-rule names.
+
+```css
+@media (max-width: 600px) {}
+/**   ↑
+ * The newline after at-rule names */
+```
+
+## Options
+
+`string`: `"always"|"always-multi-line"`
+
+### `"always"`
+
+There *must always* be a newline after at-rule names.
+
+The following patterns are considered warnings:
+
+```css
+@charset"UTF-8";
+```
+
+```css
+@charset "UTF-8";
+```
+
+```css
+@charset 
+
+"UTF-8";
```

Comment:
I guess do whatever is consistent with the existing rules, and create a new issue if we need to reconsider.